src/avif.c

```diff
@@ -101,6 +101,20 @@ const char * avifResultToString(avifResult result)
     return "Unknown Error";
 }
 
+const char * avifProgressiveStateToString(avifProgressiveState progressiveState)
+{
+    // clang-format off
+    switch (progressiveState) {
+        case AVIF_PROGRESSIVE_STATE_UNAVAILABLE: return "Unavailable";
+        case AVIF_PROGRESSIVE_STATE_AVAILABLE:   return "Available";
+        case AVIF_PROGRESSIVE_STATE_ACTIVE:      return "Active";
+        default:
+            break;
+    }
+    // clang-format on
+    return "Unknown";
```

Comment:
Move this return statement to the default case in the switch statement.

Reply:
This is the same layout as `avifPixelFormatToString()`. If we want to change this pattern in all functions, let's do it in a separate PR.